language/move-lang/src/cfgir/cfg.rs

```diff
@@ -140,6 +139,64 @@ impl<'a> CFG for BlockCFG<'a> {
     }
 }
 
+const DEAD_ERR_CMD: &str =
+    "Unreachable code. This statement (and any following statements) will \
+     not be executed. In some cases, this will result in unused resource values.";
+
+const DEAD_ERR_EXP: &str = "Invalid use of a divergent expression. The code following the \
+evaluation of this expression will be dead and should be removed. In some cases, this is necessary \
+to prevent unused resource values.";
```

Comment:
Rust formatting between these two seem inconsistent. Any way to fix?

Reply:
I'll play around with the `\`